db/migrate/20160810142635_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string :content
+      t.integer :user_id
+      t.integer :post_id
+
```

Comment:
This also needs `t.string :post_type`. Also everything on a comment should be 'null: false`